sdks/wasm/docs/getting-started/packager.md

```diff
@@ -0,0 +1,178 @@
+        type: bool  default: false
+  --binding (enable the binding engine)
+        type: bool  default: true
+  --link-icalls (link away unused icalls)
+        type: bool  default: false
+  --il-strip (strip IL code from assemblies in AOT mode)
+        type: bool  default: true
+  --linker-verbose (set verbose option on linker)
+        type: bool  default: false
+
+```
+
+## Interpreted:
+
+Steps of execution are:
+
+1. Setup options passed from the command line.
+1. Identify the assemblies referenced by the assemblies passed to packager.
+1. Write all referenced assemblies to the output folder “managed” by default
+1. Copy the Mono WebAssembly implementations to the output folder.  Default is `release` but can be controlled by adding the —debugrt command line option.
+1. Generate a runtime.js and mono-config.js which handles the loading of Mono WebAssembly modules.
+
+### Example 1: Using packager.exe with a non AOT project
+
+``` bash
+mono packager.exe --copy=always --out=./publish --asset=./sample.html --asset=server.py sample.dll
+```
+
+Additional assets can be included with the `—asset` command line option.  These will be copied to the output folder that is created.
+
+```
+--asset=./sample.html --asset=server.py
+```
+
+The following ouput directory structure is created:
+
+```
+
+         |--- publish               // Output from the `packager.exe` application 
+            |--- managed            // Where to find the managed assemblies generated by packager.exe
+            |--- mono-config.js     // Configuration file referenced in `sample.html` and `runtime.js`
+            |--- mono.js            // Mono WebAssembly implementations
+            |--- mono.wasm          // Mono WebAssembly implementations
+            |--- runtime.js         // Handles the loading of the Mono WebAssembly modules
+            |--- sample.html        // sample html file
+            |--- server.py          // python test server
+
+```
+
+With a non AOT project the resolved assemblies are not as of yet run through the linker but instead copied directly to the managed directory.
+
+### Example 2:  Using packager.exe with a non AOT project
+
+```
+mono packager.exe  --copy=ifnewer --out=publish --search-path=./managed/ --asset=index.html --asset=RunClass.txt  ./bin/WasmRoslyn.dll
+```
+
+The example here uses the `—search-path` command line option.  The `—search-path` will add another path that will be used to resolve assemblies that are identified in the specified `WasmRoslyn.dll` assembly.
+
+Again, no linking of the the assemblies is as of yet implemented for non AOT’d projects.
+
+## AOT:
+
+AOT support is enabled by passing `--aot` to the packager.  It requires a functional emscripten sdk installation.
+
+Access to emscripten sdk which is controlled by:
+
+```
+--emscripten-sdkdir
+```
+
+Access to mono sdk which is controlled by:
+
+```
+-mono-sdkdir=
+```
+
+Steps of execution are:
+
+1. Setup options passed from the command line.
```

Comment:
Not sure this needs to be in a 'getting started' document.

Reply:
I was wondering the same after I already committed it.  I will move it directly to the `docs` directory

Reply:
Addressed in latest commit.  Moved to docs directory and a link to the main WebAssembly readme